src/benchmark.cc

```diff
@@ -699,7 +743,11 @@ void RunBenchmark(const benchmark::internal::Benchmark::Instance& b,
         report.report_label = label;
         // Report the total iterations across all threads.
         report.iterations = static_cast<int64_t>(iters) * b.threads;
-        report.real_accumulated_time = real_accumulated_time;
+        if (b.use_manual_time) {
```

Comment:
do you think it's worth separating out manual accumulated time from real accumulated time? both could be set on a benchmark, and we already have both CPU and real accumulated time.